### PR TITLE
[BUG] Https protocol schema is not handled correctly in java 11 nativ…

### PR DESCRIPTION
**Pull request by thecrazzymouse**
…e code #3729

 * added getter for schema

### CODE REVIEW

**Comment by macjohnny** [comment]
cc @bbdouglas

**Comment by macjohnny** [comment]
@thecrazzymouse thanks for the PR.
could you please re-generate the samples and commit them? see https://github.com/OpenAPITools/openapi-generator/blob/master/.github/PULL_REQUEST_TEMPLATE.md

File: modules/openapi-generator/src/main/resources/Java/libraries/native/ApiClient.mustache [hunk 1]
```diff
@@ -252,6 +252,26 @@ public class ApiClient {
...
   }
 
+  /**
+   * Set a custom schema for the target service, for example 'https'.
+   *
+   * @param schema The schema of the target service
+   * @return This object.
+   */
+  public ApiClient setSchema(String schema){
+    this.schema = schema;
```
Comment (macjohnny) [inline modules/openapi-generator/src/main/resources/Java/libraries/native/ApiClient.mustache]: I think it should be `scheme`, not `schema`

Comment (thecrazzymouse) [reply modules/openapi-generator/src/main/resources/Java/libraries/native/ApiClient.mustache]: Fixed the typo

**Comment by macjohnny** [comment]
@thecrazzymouse could you please re-generate the samples and commit them? see https://github.com/OpenAPITools/openapi-generator/blob/master/.github/PULL_REQUEST_TEMPLATE.md

**Comment by macjohnny** [comment]
ci failure is unrelated (kotlin samples)

**Comment by wing328** [comment]
@thecrazzymouse thanks for the PR, which has been included in the v4.1.1 release: https://twitter.com/oas_generator/status/1165944867391860737

**Pull request merged**